content/guides/core/automations/_index.md

```diff
@@ -6,4 +6,36 @@ menu:
 title: Automations
 weight: 4
 ---
+{{% pageinfo color="info" %}}
+{{< readfile file="/_includes/enterprise-only.md" >}}
+{{% /pageinfo %}}
 
+This page describes _automations_ in W&B. [Create an automation]({{< relref "create-automations/" >}}) to trigger workflow steps, such as automated model testing and deployment, based on an event in W&B, such as when an [artifact]({{< relref "/guides/core/artifacts" >}}) artifact version is created ({{< relref "/guides/core/registry/" >}}).
```

Comment:
Nit: Add a definition after first mention of automations, following the italicization of the word.

Reply:
I think that's what the second sentence (after the italics) aims to do. If it's OK, I will leave it as is for now?